docs/deployment-guide.md

```diff
@@ -708,11 +708,21 @@ In that case, you can add the `cluster_keepoff` variable to one of the `reposito
 
 > A typical use case is to have a dedicated Solr tracking node. The playbook will then prefer to use that dedicated node - if it finds one - for solr tracking and only use the other as backup server (no load balancing)
 
-## Cleanup
+## Cleanup & Uninstallation of ACS 
 
-What needs to be removed from a system will depend on your inventory configuration. The steps below presume a localhost or single machine deployment i.e. where all roles were run on the same machine.
+What needs to be removed from a system will depend on your inventory configuration. The steps below presume a cleanup and uninstallation of Alfresco content service after deployment of ansible artifacts by using platform-cleanup.yml playbook and platform-uninstall.yml playbook respectively. 
 
-1. Stop and remove the following systemd services:
+# Cleanup
+
+This playbook will remove the temporary artifacts which are stored on the hosts.In order to cleanup the system post deployment run the following command:
```

Comment:
Can you add a reminder about the fact running that playbook breaks playbook idempotency?